InventoryComponents/InventoryConsumablesComponent.gd

```diff
@@ -43,14 +43,16 @@ func use_consumable_at_place(pos : int) -> void:
 	_held_consumables[pos].on_consume()
 	remove_consumable_at_place(pos)
 
-## fills the _held_consumables array with empty slots or removes slots if there are too many
-## If there is a consumable in the slot that wil be removed it will dissapear
+## fills the _held_consumables array with empty slots or removes slots if there are too many [br]
+## If there is a consumable in the slot that wil be removed the game will try to give it to the player [br]
+## as if it was a new item, but if there are no open slots it will dissapear
```

Comment:
do you wan this to show on two separate lines ?
basically the [br] is used to tell that the text should display on two separate lines. If you don't say anything, there won't be newlines. Usually, a single sentence should have a break in the middle, because it's a bit
strange to have a sentence that looks like that

Reply:
I actually don't understand how [br] works should I write###  If there is a consumable in the slot that wil be removed the game will try to give it to the player [br] as if it was a new item, but if there are no open slots it will dissapear in one line?

Reply:
The best way to understand is to open the doc on the Godot editor.
If you write :
```
## If there is a consumable in the slot that wil be removed the game will try to give it to the player [br]
## as if it was a new item, but if there are no open slots it will dissapear
```
this will display in the doc as: 
```
If there is a consumable in the slot that wil be removed the game will try to give it to the player
as if it was a new item, but if there are no open slots it will dissapear
```
If you write:
```
## If there is a consumable in the slot that wil be removed the game will try to give it to the player
## as if it was a new item, but if there are no open slots it will dissapear
```
It will display as:
```
If there is a consumable in the slot that wil be removed the game will try to give it to the player as if it was a new item, but if there are no open slots it will dissapear
```
If you write:
```
## If there is a consum[br]able in the slot that wil be removed the game will try to give it to the player
## as if it was a new item, but if there are no open slots it will dissapear
```
it will display as:
```
If there is a consum
able in the slot that wil be removed the game will try to give it to the player as if it was a new item, but if there are no open slots it will dissapear
```
And if you write:
```
## If there is a consumable in the slot that wil be removed the game will try to give it to the player as if it was a new item, but if there are no open slots it will dissapear
```
It will display as:
```
If there is a consumable in the slot that wil be removed the game will try to give it to the player as if it was a new item, but if there are no open slots it will dissapear
```

The [br] tag is to signify to the doc where to go back to the line. The actual way in which you go back to the line when writing the doc doesn't matter. Return to the line are just controlled by [br] tags
Btw, it should be "slot that wil**l** be removed" and `will **disappear**`

Reply:
Ok, I think I get it

Reply:
Is there a problem with this line though? it seems fine to me

Reply:
Well the problem is that the single sentence will be displayed on two lines. As I said, open the Godot editor and check for yourself, it's the easiest way to see